registry/api/v2/errors.go

```diff
@@ -24,7 +24,7 @@ var (
 		Description: `The access controller denied access for the operation on
 		a resource. Often this will be accompanied by a 401 Unauthorized
 		response status.`,
-		HTTPStatusCode: http.StatusForbidden,
+		HTTPStatusCode: http.StatusUnauthorized,
```

Comment:
401 seems correct (403 indicates that authenticating still won't permit access) but we should confirm the engine expects and behaves correctly to 401 in this context.